packages/connection-form/src/components/advanced-options-tabs/authentication-tab/authentication-tab.tsx

```diff
@@ -43,13 +42,6 @@ interface TabOption {
 }
 
 const options: TabOption[] = [
-  {
-    title: 'None',
-    id: 'AUTH_NONE',
-    component: function None() {
-      return <></>;
-    },
-  },
```

Comment:
I do think we probably want to mark the username/passwords fields as `Optional` now (unconditionally, because we don’t really adjust that part of the form representation depending on other state in the same form)

Reply:
thought the same, i've asked in the design channel